src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/IdeaCollection/Workbench/Module.ts

```diff
@@ -27,6 +27,10 @@ export var register = (angular) => {
             AdhResourceAreaModule.moduleName,
             AdhTopLevelStateModule.moduleName
         ])
+        // the (document-focused) IdeaCollection is also registered under the directive name
+        // "adhDebateWorkbench" such as not to break currently running embeds.
+        .directive("adhDebateWorkbench", [
+            "adhTopLevelState", "adhConfig", "adhHttp", Workbench.workbenchDirective])
```

Comment:
A better alternative may be to register an embed alias (see the definition of `registerDirective()` in Embed.ts)